src/modules/fundingManager/bondingCurve/abstracts/BondingCurveBase_v1.sol

```diff
@@ -342,8 +345,23 @@ abstract contract BondingCurveBase_v1 is IBondingCurveBase_v1, Module_v1 {
         if ((_protocolFee + _workflowFee) > BPS) {
             revert Module__BondingCurveBase__FeeAmountToHigh();
         }
-        protocolFeeAmount = _totalAmount * _protocolFee / BPS;
-        workflowFeeAmount = _totalAmount * _workflowFee / BPS;
+        // Calculate protocol fee amount if applicable
+        if (_protocolFee > 0) {
+            protocolFeeAmount = _totalAmount * _protocolFee / BPS;
+        }
+        // Calculate workflow fee amount if applicable
+        if (_workflowFee > 0) {
+            workflowFeeAmount = _totalAmount * _workflowFee / BPS;
+        }
+
+        // Revert if calculated protocol/workflow fee amount rounded down to zero
+        if (_protocolFee > 0 && protocolFeeAmount == 0) {
+            revert Module__BondingCurveBase__BuyAmountToLow();
```

Comment:
What about sell?
Isnt this function going in both directions?

Reply:
You're right. Changing it